src/vcpkg/commands.ci.cpp

```diff
@@ -74,6 +77,53 @@ namespace
     private:
         Path base_path;
     };
+
+    enum class CiBaselineState
+    {
+        Skip,
+        Fail,
+    };
+
+    struct CiBaselineLine
+    {
+        std::string port_name;
+        std::string triplet_name;
+        CiBaselineState state;
+    };
+
+    std::vector<CiBaselineLine> parse_ci_baseline(View<std::string> lines)
+    {
+        std::vector<CiBaselineLine> result;
+        for (auto& line : lines)
+        {
+            if (line.empty() || line[0] == '#') continue;
+            CiBaselineLine parsed_line;
+
+            auto colon_loc = line.find(':');
+            Checks::check_exit(VCPKG_LINE_INFO, colon_loc != std::string::npos, "Line '%s' must contain a ':'", line);
+            parsed_line.port_name = Strings::trim(StringView{line.data(), line.data() + colon_loc}).to_string();
+
+            auto equal_loc = line.find('=', colon_loc + 1);
+            Checks::check_exit(VCPKG_LINE_INFO, equal_loc != std::string::npos, "Line '%s' must contain a '='", line);
+            parsed_line.triplet_name =
+                Strings::trim(StringView{line.data() + colon_loc + 1, line.data() + equal_loc}).to_string();
```

Comment:
FYI I'm changing this behavior because it fails, for example, with

```
catch-classic:arm64-windows      = skip
catch-classic:arm-uwp            = skip
catch-classic:x64-linux          = skip
catch-classic:x64-osx            = skip
catch-classic:x64-uwp            = skip
catch-classic:x64-windows        = skip
catch-classic:x64-windows-static = skip
catch-classic:x64-windows-static-md=skip
catch-classic:x86-windows        = skipcatch-classic:arm64-windows      = skip
catch-classic:arm-uwp            = skip
catch-classic:x64-linux          = skip
catch-classic:x64-osx            = skip
catch-classic:x64-uwp            = skip
catch-classic:x64-windows        = skip
catch-classic:x64-windows-static = skip
catch-classic:x64-windows-static-md=skip
catch-classic:x86-windows        = skip
```